airflow/example_dags/plugins/workday.py

```diff
@@ -78,6 +78,9 @@ def next_dagrun_info(
 
     # [END howto_timetable_next_dagrun_info]
 
+    def validate(self):
+        pass
```

Comment:
Doesn’t the base class provide a default (empty) implementation?

Reply:
I think it does, by new version of mypy doesn't seem to see the impl from the Protocol baseclass for some reason (arguably a bug in mypy)

Reply:
(GitHub needs a 🤦 reaction)